### ISSUE TITLE
everit-json-schema was replaced by erosb/json-sKema

### ISSUE THREAD

**Issue by ieugen** [luposlip/json-schema#16]
Hi,

It seems the author dropped the original project to focus on a new one https://github.com/erosb/json-sKema .
See deprecation notice: https://github.com/everit-org/json-schema?tab=readme-ov-file#deprecation-notice

**Comment by luposlip** [comment]
Yeah, that's right. The author of #13 haven't had time to look at it, and unfortunately, so haven't I.

Personally I haven't had any use cases that requires supporting any JSON schema version after Draft-7. But I'll look into this eventually, if noone else have the courage to give it a go before that 😄 Realistically I won't have the time until either this summer, or summer 2025.

**Comment by ieugen** [comment]
thanks. 

I made the issue more for tracking. 
I cam to json-schema for parsing compose-spec but I need it to be compatible with babashka.
Which means no java libs :(

**Comment by luposlip** [comment]
Yeah, that's a valid use case. Noted, and thanks for the input!

lør. 16. mar. 2024 kl. 12.52 skrev Eugen Stan ***@***.***>:
